src/main/java/seedu/address/model/tag/LevelSubjectCode.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.tag;
+
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public enum LevelSubjectCode {
```

Comment:
May also want to check if all usages of the tag-related static constants in `CommandTestUtil.java` tally with this scope of a valid tag.